### ISSUE TITLE
TypeScript version warning on a new Vue project

### ISSUE THREAD

**Issue by vvnsrzn** [vuejs/eslint-config-typescript#75]
Hi there,

On a new vue project, I'm getting this warning :

```
=============

WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree.

You may find that it works just fine, or you may not.

SUPPORTED TYPESCRIPT VERSIONS: >=4.3.5 <5.4.0

YOUR TYPESCRIPT VERSION: 5.4.3

Please only submit bug reports when using the officially supported version.

=============
```

Do you know how can I get rid of it?

Should I bump vuejs/eslint-config-typescript to v13?

### Details:
OS : Ubuntu 18 & 20
Node : 20.11.1

### Steps to reproduce:
```
$ npm create vue (pick TypeScript + EsLint)
$ cd vue-project
$ npm install
$ npm run lint
```

Thanks for your help!
